src/components/color-picker-hex-input/color-picker-hex-input.tsx

```diff
@@ -42,6 +41,13 @@ export class ColorPickerHexInput implements LoadableComponent {
 
   @Element() el: HTMLCalciteColorPickerHexInputElement;
 
+  /**
+   * Specifies accessible label for `calcite-input`.
+   *
+   * @readonly
+   */
+  @Prop() hexLabel = "Hex";
```

Comment:
Maybe we can just leave it as empty string?

Reply:
leaving this empty would break `a11y` test in `color-picker-hex-input`  with `label` being empty.

Reply:
Ok. this is fine then.